### PR TITLE
#159989010 setup sass

### PR DESCRIPTION
**Pull request by klevamane**
#### What does this PR do?
Setup sass preprocessing on the application
#### Description of Task to be completed?
- add sass-loader to webpack
- configure webpack to test for .scss
#### How should this be manually tested?
In ComandLine:
- `git clone https://github.com/andela/fargo-ah-client.git Authors-Haven-Client`
- `npm install`
- `add sass or css styles to style.scss file in /src/scss/style.scss`
-  `import the style.scss file in the desired component .js file`
- `npm run start` the start script

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#159989010

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ module.exports = {
               minimize: true,
             },
           },
+          {
+            loader: 'sass-loader', // compiles Sass to CSS
```
Comment (Oloyedesinmiloluwa) [inline webpack.config.js]: please re-write this comment to follow Airbnb comment style https://github.com/airbnb/javascript#comments

**Review by Oloyedesinmiloluwa — COMMENTED** [review]
Please write a minimalistic code in the `.scss` file to show it is actually bundling

**Review by anneKay — CHANGES_REQUESTED** [review]
Please use the requested format for naming pull request.

**Review by anneKay — APPROVED** [review]

**Review by Oloyedesinmiloluwa — APPROVED** [review]
Alright, I think this is ok to be merged

**Review by anuonifade — APPROVED** [review]
LGTM

**Pull request merged**